emulator/contracts.go

```diff
@@ -14,8 +14,7 @@ import (
 	nftstorefront "github.com/onflow/nft-storefront/lib/go/contracts"
 )
 
-var CommonContracts = func() []ContractDescription {
-	chain := flowgo.Emulator.Chain()
+var CommonContractsGenerator = func(chain flowgo.Chain) []ContractDescription {
```

Comment:
Maybe just declare as a function now, and use `New*` naming scheme:

```suggestion
func NewCommonContracts(chain flowgo.Chain) []ContractDescription {
```

Reply:
great idea